### PR TITLE
add html&css practice task: popup

### PR DESCRIPTION
**Pull request by AMashoshyna**
As discussed in chat, here is a proposal of additional task for peer-to-peer course.

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
@AMashoshyna Great idea! Students can use the learning when completing Frogger and MPG.

Please, consider changing [course contents](https://github.com/kottans/frontend/blob/master/contents.md) appropriately.

**Comment by OleksiyRudenko** [comment]
@AMashoshyna 
Oh, have forgotten about `./tasks/README.md`
I meant rather https://github.com/kottans/frontend/blob/master/contents.md

`./tasks/README.md` is rather a working collection and so much exposed to the outer world.

Let's update both.

`./tasks/README.md` will die once we find a way how to properly list **all** tasks (offline main and offline supplementary) to `./contents.md`

Thanks!

**Comment by AMashoshyna** [comment]
@yevhenorlov @A-Ostrovnyy could you please take a look?

File: tasks/html-css-popup.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[![MIT Licensed][icon-mit]][license]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Telegram][icon-chat]][chat]
+
+# HTML & CSS practice: Google-style Popup
+
+The goal of this task is to practice coding HTML & CSS from mockup. You are expected to deepen your knowledge of HTML & SCC and discover new previously unknown features.
```
Comment (A-Ostrovnyy) [inline tasks/html-css-popup.md]: HTML & CSS

File: tasks/html-css-popup.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Requirements:
+- no JavaScipt
```
Comment (A-Ostrovnyy) [inline tasks/html-css-popup.md]: `no JavaScript, only HTML/CSS use of preprocessors allowed.`
How you look on that?

File: tasks/html-css-popup.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+![popup-visible](../img/popup-visible.png)
+
+Scrollable state:
+
+![popup-scrollable](../img/popup-scrollable.png)
+
+All interactive elements should be marked as such.
+
+![popup-hover-state](../img/popup-hover.png)
+
```
Comment (A-Ostrovnyy) [inline tasks/html-css-popup.md]: `All interactive elements should have :focus state.`
Add image with `:focus` state (underscored caption).

File: tasks/html-css-popup.md (hunk 1 above)
Comment (AMashoshyna) [reply tasks/html-css-popup.md]: oops)

File: tasks/html-css-popup.md (hunk 3 above)
Comment (AMashoshyna) [reply tasks/html-css-popup.md]: nice, thanks

File: tasks/html-css-popup.md (hunk 2 above)
Comment (AMashoshyna) [reply tasks/html-css-popup.md]: sure, good point

**Pull request merged**